packages/web-extension/src/keyAgent/util.ts

```diff
@@ -6,6 +6,7 @@ export const keyAgentChannel = (walletName: string) => `${walletName}$-keyAgent`
 export const keyAgentProperties: RemoteApiProperties<AsyncKeyAgent> = {
   deriveAddress: RemoteApiPropertyType.MethodReturningPromise,
   derivePublicKey: RemoteApiPropertyType.MethodReturningPromise,
+  extendedAccountPublicKey: RemoteApiPropertyType.MethodReturningPromise,
```

Comment:
I think this belongs in f905f2a72d8c0707bd18cd53d49d41503cb12ff7

Reply:
I will extract this before squashing.

Reply:
done